test/functionalTest/cases/3842_per_sub_httpTimeout/per_sub_httpTimeout_crud.test

```diff
@@ -0,0 +1,421 @@
+# Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Per subscrition httpTimeout CRUD
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. Create a subscrition with a correct httpTimeout field, get 204
+# 02. GET /v2/subscriptions/ID
+# 03. Modify the subscription changing httpTimeout, get 204
+# 04. GET /v2/subscriptions/ID
```

Comment:
I'd suggest to include also this case:

```
05. Modify the subscription removing timeout, get 204
06. GET /v2/subscriptions/ID and see no timeout is field is foud
07. Modify the subscription to include again a timeout
08. GET /v2/subscriptions/ID and see the added timeout
```

(current steps >05 should be renumbered)